include/igl/pinv.h

```diff
@@ -12,6 +12,9 @@ namespace igl
   // Outputs:
   //   X  n by m matrix so that A*X*A = A and X*A*X = X and A*X = (A*X)' and
   //     (X*A) = (X*A)'
+  //
+  // Obsolete: Use Eigen::CompleteOrthogonalDecomposition<Eigen::MatrixXd>
```

Comment:
Mark the function as `[[deprecated]]` then?

Reply:
nice. I didn't know we had this deprecated.h